src/App.js

```diff
@@ -0,0 +1,28 @@
+import NavBar from "./components/NavBar";
```

Comment:
Imports should in alphabetical order, to avoid doing this conciously you can configure your VS Code to do so.